src/core/gosling.schema.guards.ts

```diff
@@ -185,6 +185,9 @@ export function IsDataDeepTileset(
     );
 }
 
+/**
+ * Checks if a channel is of type ChannelDeep
+ */
 export function IsChannelDeep(channel: ChannelDeep | ChannelValue | undefined): channel is ChannelDeep {
     return typeof channel === 'object' && !('value' in channel);
```

Comment:
Just having a look through is. I think `isObject` is now a gaurd in this module.

```suggestion
    return isObject(channel) && !('value' in channel);
```